src/project-model.js

```diff
@@ -0,0 +1,38 @@
+function createProject(title) {
+  const myTitle = title;
+  const todos = [];
+
+  const getTitle = () => myTitle;
+
+  const addTodo = (todo) => {
+    todos.push(todo);
+  };
+
```

Comment:
- [ ] Kindly you need to use local storage to save your to-do items and projects. This is a hard requirement for this project as it will enable you to learn how to use local storage and also persist your application data so that it isn't lost when the page refreshes. You can learn more about local storage using this [article](https://www.w3schools.com/jsref/prop_win_localstorage.asp) ⬇️ 
![localstorage_requirement](https://user-images.githubusercontent.com/32361963/117766963-03b4e980-b239-11eb-9902-b5729a16e836.png)